lib/tasks/ovirt.rake

```diff
@@ -14,9 +14,8 @@ namespace :ovirt do
       # OVIRT_PRETEND=true foreman-rake ovirt:drop
   END_DESC
 
-  task :drop => :environment do
-    compute_resources = ComputeResource.unscoped.where(type: 'Foreman::Model::Ovirt')
-
+  task drop: :environment do
```

Comment:
With oVirt gone and a DB migration to drop thrn, is there any point in keeping the task?

Reply:
:thinking: I would have to check if we really cherry-picked it to all older and supported Foreman and Satellite versions.

Reply:
So,
It's part of the 3.15, and for the downstream, I created the https://issues.redhat.com/browse/SAT-34779.

Regarding the removal, there may be one non-standard scenario where it could be helpful.
Let's say users will have an issue during the `foreman-installer` run, due to the migration file.

What they might do:
* (temporarily) remove the migration file
* Run the rake task
* Fix the issues unless the rake task passes
* Profit

One advantage is that rake task is faster to run and verify, while the installer process is usually slower and takes more time.